asv_bench/benchmarks/pandas/arithmetic.py

```diff
@@ -0,0 +1,379 @@
+import operator
+import warnings
+
+import numpy as np
+
+import modin.pandas as pd
+from modin.pandas import (
+    DataFrame,
+    Series,
+    date_range,
+)
+
+from .pandas_vb_common import numeric_dtypes
+
+import pandas.tseries
+from pandas.tseries import offsets
+try:
+    import pandas.tseries.holiday
+except ImportError:
+    pass
+
+
+class IntFrameWithScalar:
+    params = [
+        [np.float64, np.int64],
+        [2, 3.0, np.int32(4), np.float64(5)],
+        [
+            operator.add,
```

Comment:
do we need all of them, really? I think we should try to pick minimum number of operations that could represent some particular piece of Modin's code